styles.css

```diff
@@ -728,3 +754,666 @@ a {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+  }
+
+  .email_icon {
+    margin-right: 20px;
+  }
+
+  .nav_link:hover {
+    color: #f55800;
+  }
+
+  .hamburger_icon {
+    display: none;
+  }
+
+  .nav_email_icon {
+    display: block;
+    width: 25px;
+    height: 3px;
+    margin: 5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #fff;
+  }
+
+  .hr-line_break3 {
+    display: none;
+  }
+
+  .mobile_nav_container {
+    display: none;
+  }
+
+  .main_section {
+    background-color: #1c1a19;
+    background-image: url('./image folder/Illustration-header.png');
+    background-position: right bottom;
+    background-repeat: no-repeat;
+    background-size: auto;
+    height: 800px;
+    padding: 0 24px;
+    color: white;
+  }
+
+  .main_section_div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 12rem 0.5rem 12rem 12rem;
+    gap: 24px;
+    width: 35.38rem;
+    height: 46.19rem;
+    left: 143px;
+    top: 72px;
+  }
+
+  .h1_title {
+    width: 35.38rem;
+    height: 10rem;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .h2_title {
+    width: 566px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 2rem;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .p_content {
+    width: 566px;
+    height: 123px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 2rem;
+    color: #f4f5f7;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .social_media_group {
+    position: absolute;
+    height: auto;
+    list-style-type: none;
+    flex-direction: column;
+    flex-basis: 0;
+    flex-shrink: 0;
+    margin-top: 19rem;
+  }
+
+  .social_media_icons {
+    margin-top: 0;
+  }
+
+  /* Recent works */
+
+  .recent_work_div {
+    display: flex;
+    flex-direction: row;
+    width: 56.25rem;
+  }
+
+  .image_line {
+    margin-top: 2.81rem;
+    width: 60%;
+  }
+
+  .recent_works_header {
+    width: 327px;
+    height: 83px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #1c1a19;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .recent_work_content {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 0;
+    gap: 2px;
+    width: 1156px;
+    height: 448px;
```

Comment:
same comment here , please don't use fixed width and heights for your containers as did will affect your design in different screen sizes